unstructured/partition/html.py

```diff
@@ -0,0 +1,38 @@
+from typing import IO, List, Optional
+
+from unstructured.documents.elements import Element
+from unstructured.documents.html import HTMLDocument
+
+
+def partition_html(filename: str = "", file: Optional[IO] = None, text: str = "") -> List[Element]:
```

Comment:
Nit: Could we use `None` as the default for arguments that weren't specified? If this sits in an automated pipeline somewhere I can imagine an empty string being passed for `filename` (in which case some kind of "invalid filename" from the file handler response would be more helpful than "One of filename, file, or text must be specified.") or for `text` (in which case we'd want to return an empty `HTMLDocument`).

Reply:
Yep! Just updated the defaults